common/src/test/scala/uk/ac/wellcome/models/MergedSierraRecordTest.scala

```diff
@@ -195,42 +196,149 @@ class MergedSierraRecordTest extends FunSpec with Matchers {
     }
 
     describe("when the merged record is unlinked from the item") {
-      ignore("should remove the item if it already exists") {
-        val unlinkedBibId = "222"
+      it("removes the item if it already exists") {
+        val bibId = "222"
+
         val record = sierraItemRecord(
           id = "i111",
           title = "Only otters occupy the orange oval",
           modifiedDate = "2001-01-01T01:01:01Z",
-          bibIds = List("i111"),
-          unlinkedBibIds = List(unlinkedBibId)
+          bibIds = List(bibId),
+          unlinkedBibIds = List()
+        )
+
+        val unlinkedItemRecord = record.copy(
+          bibIds = Nil,
+          modifiedDate = record.modifiedDate.plusSeconds(1),
+          unlinkedBibIds = List(bibId)
+        )
+
+        val mergedSierraRecord = MergedSierraRecord(
+          id = bibId,
+          itemData = Map(record.id -> record)
+        )
+
+        val expectedMergedSierraRecord = mergedSierraRecord.copy(
+          itemData = Map.empty
         )
-        val mergedSierraRecord = MergedSierraRecord(id = unlinkedBibId,
-                                                    itemData =
-                                                      Map(record.id -> record))
-        mergedSierraRecord.unlinkItemRecord(record) shouldBe mergedSierraRecord
-          .copy(itemData = Map.empty)
+
+        mergedSierraRecord.unlinkItemRecord(unlinkedItemRecord) shouldBe expectedMergedSierraRecord
       }
 
-      ignore(
-        "should return the original record when merging an unlinked record which is already absent") {
-        true shouldBe false
+      it("returns the original record when merging an unlinked record which is already absent") {
+        val bibId = "333"
+
+        val record = sierraItemRecord(
+          id = "i111",
+          title = "Only otters occupy the orange oval",
+          modifiedDate = "2001-01-01T01:01:01Z",
+          bibIds = List(bibId),
+          unlinkedBibIds = List()
+        )
+
+        val previouslyUnlinkedRecord = sierraItemRecord(
+          id = "i222",
+          title = "Only otters occupy the orange oval",
+          modifiedDate = "2001-01-01T01:01:01Z",
```

Comment:
Want to adjust the modified date so it’s definitely later than the previous record?

Reply:
I think all the dates can be cleaned up here - though i'm not sure this one should be changed, the test really should be "ignore an unrelated unlinking event".